main.nf

```diff
@@ -237,11 +244,11 @@ if(!params.bismark_index && params.fasta && params.aligner == 'bismark'){
  */
 if(!params.bwa_meth_index && params.fasta && params.aligner == 'bwameth'){
     process makeBwaMemIndex {
-        tag fasta
+        tag "$fasta"
         publishDir path: "${params.outdir}/reference_genome", saveAs: { params.saveReference ? it : null }, mode: 'copy'
 
         input:
-        file fasta from fasta
+        file fasta from fasta_1
```

Comment:
Can you please use names that are more explicit?

ch_from_blabla_to_blabla is something that helps you (and me, others) to keep things traceable :-)